cmd/webpkgserver/webpkgserver.example.toml

```diff
@@ -202,6 +202,89 @@
   # bytes for the OCSP response.
   #AllowTestCert = false
 
+# IMPORTANT NOTE: the support of the ACME protocol and automatic renewal of
+# certificates is currently in the EXPERIMENTAL stage.  Once we have more
+# experience with people using it out in the wild, we will gradually move it to
+# the PRODUCTION stage.
+#
+# This section is meant for ACME config, which allows the Web Packager to use
+# ACME to automatically renew Signed Exchange certificates.
```

Comment:
"to use ACME" doesn't look to give much information (because it's implied by "This section is meant for ACME config" or "Configure ACME").

Reply:
Done.